src/SFA.DAS.Provider.PR.Web/Controllers/EmployerDetailsController.cs

```diff
@@ -29,4 +29,31 @@ public async Task<IActionResult> Index([FromRoute] int ukprn, [FromRoute] string
 
         return View(model);
     }
+
+    [Route("{ukprn:int}/employers/{requestid:Guid}", Name = RouteNames.EmployerDetailsByRequestId)]
+    [HttpGet]
+    public async Task<IActionResult> Index([FromRoute] int ukprn, [FromRoute] Guid requestid,
+        CancellationToken cancellationToken)
+    {
+        GetRequestsByRequestIdResponse response;
+        try
+        {
+            response = await _outerApiclient.GetRequestByRequestId(requestid, cancellationToken);
```

Comment:
I have suggested changes in the client that will allow you to inspect response status, which if it is 404 then we can redirect the user accordingly, no need for try catch